vresutils/decorators.py

```diff
@@ -66,9 +66,16 @@ def cachable(func=None, version=None, cache_dir=config['cache_dir'],
                   True).
     """
     enable = os.path.isdir(cache_dir)
+    
+    def root_dir(p):
+        pn = os.path.dirname(p)
+        if pn == p:
+            return pn
+        else:
+            return root_dir(pn)
 
     if enable:
-        st = os.stat(cache_dir[0])
+        st = os.stat(root_dir(cache_dir))
```

Comment:
```suggestion
        st = os.stat(cache_dir)
```

Reply:
The code in the `if enable` clause was supposed to get the unix permissions of the cache directory, so that later on we can set the same permissions on the cache files.

I mistakenly introduced the `[0]` back when I added the `fallback_cache_dirs` (https://github.com/FRESNA/vresutils/commit/80c9997c6060394738a83fd65a44ffc9e6d07bc6). In some intermediate state `cache_dir` was a list of cache directories and then you wanted the first one from the list.